tests/validation/test_transaction_structure.py

```diff
@@ -66,30 +66,30 @@ def test_tx_serialization_with_no_hash(signed_create_tx):
 ################################################################################
 # Operation
 
-def test_validate_invalid_operation(b, create_tx):
+def test_validate_invalid_operation(b, create_tx, alice):
```

Comment:
I think all these transactions can be enabled now i.e. `pytestmark = pytest.mark.tendermint`?